lib/mux_tf/yaml_cache.rb

```diff
@@ -1,5 +1,11 @@
 require "yaml/store"
 
+module YAML
```

Comment:
Please add some comments as to the why of this

Reply:
It's to override the way that psych seems to be loading YAML. instead of using 'load' which needs work to permit the 'Time' class (which from above I tried that and it wasn't working so I decided to just bypass and use what it was doing before). This brings us back to the equivalent that was working before in that unsafe load was used before the psych upgrade. I'm not sure how much more I can go into the why... the ref above is the changes that 'cause the problem' and so I'm 'fixing it' by using the old equivalent. Maybe the yaml cache needs more work to have `YAML::Store.new(path, { aliases: true, permitted_classes: [Time] })` work.